Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -4090,6 +4258,511 @@ def qradar_remote_network_deploy_execution_command(client: Client, args):
+      protocol parameter.
+    - descrption: The description of the log source
+    - coalesce_events: Determines if events collected by this log source are coalesced based on common properties.
+      If each individual event is stored, then the condition is set to false. Defaults to true.
+    - enabled: Determines if the log source is enabled. Defaults to true.
+    - parsing_order: The order in which log sources will parse if multiple exists with a common identifier.
+    - group_ids: The set of log source group IDs this log source is a member of.
+      Each ID must correspond to an existing log source group.
+      See the Log Source Group API (https://ibmsecuritydocs.github.io/qradar_api_20.0/20.0--config-event_sources-log_source_management-log_source_groups-id-GET.html).
+    - credibility: On a scale of 0-10, the amount of credibility that the QRadar administrator places on this log source
+    - store_event_payload: If the payloads of events that are collected by this log source are stored, the condition is set to
+      'true'. If only the normalized event records are stored, then the condition is set to 'false'.
+    - target_event_collector_id:  The ID of the event collector where the log source sends its data.
+      The ID must correspond to an existing event collector.
+    - disconnected_log_collector_id:  The ID of the disconnected log collector where this log source will run.
+      The ID must correspond to an existing disconnected log collector.
+    - language_id: The language of the events that are being processed by this log source.
+      Must correspond to an existing log source language.
+    - requires_deploy: Set to 'true' if you need to deploy changes to enable the log source for use;
+      otherwise, set to 'false' if the log source is already active.
+    - wincollect_internal_destination_id : The internal WinCollect destination for this log source, if applicable.
+      Log sources without an associated WinCollect agent have a null value. Must correspond to an existing WinCollect destination.
+    - wincollect_external_destination_ids: The set of external WinCollect destinations for this log source, if applicable.
+      Log Sources without an associated WinCollect agent have a null value.
+      Each ID must correspond to an existing WinCollect destination.
+    - gateway: If the log source is configured as a gateway, the condition is set to 'true';
+      otherwise, the condition is set to 'false'. A gateway log source is a stand-alone protocol configuration.
+      The log source receives no events itself, and serves as a host for a protocol configuration that retrieves event data to
+      feed other log sources. It acts as a "gateway" for events from multiple systems to enter the event pipeline.
+    """
+    id = args.get('id')
+    pp_pairs = args.get('protocol_parameters').split(',') if args.get('protocol_parameters') else None
```

Comment:
```suggestion
    pp_pairs = args.get('protocol_parameters').split(',') if args.get('protocol_parameters') else []
```
better practice to not treat empty cases special

Reply:
done